### DISCUSSION TITLE
See the replays I've commented on

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#2405] [Ideas]
I'm often in the spot where i am looking for a replay that i've commented on. In these cases, it might not even appear in my library because i'm likely not the author or collaborator.

**Comment by jonbell-lot23** [comment]
100%. The design I’m working towards is one with a series of simple dropdowns across the top that let you see the query you want, and this would be one of the options.

I didn’t realise this was a bug so I’ll prioritise it.

**Comment by jaril** [comment]
It's a valid use case that will pop up over and over again. There should be a way to surface replays based on my recent activity (viewing a replay, leaving a comment). It will probably be a simple reverse chronological list, since recency will be highly correlated with being the thing that the user is looking for.

There's a few ways to tackle it, but I'll defer to @jonbell-lot23 since all of them could conflict with the spreadsheet filter view we're moving towards.

**Reply by jonbell-lot23** [reply]
Bingo. I wouldn’t say they *all* conflict, I’d say there’s the Microsoft or Google Drive way of handling this, and we have ideas that are much more clear.

Fortunately we’ve discussed this a bunch so now I’ll built out that design.

**Comment by jonbell-lot23** [comment]
Hm, how do I turn a discussion into an issue? I guess that's not possible?

**Comment by jonbell-lot23** [comment]
Filed: https://github.com/RecordReplay/devtools/issues/2406

**Comment by jonbell-lot23** [comment]
We did ship this, then removed it back out again. We may bring it back as part of pagination. cc @ryanjduffy
